apps/www/_blog/2023-11-01-supabase-is-now-compatible-with-nextjs-14.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Supabase is now compatible with Next.js 14
+description: The fastest way to build apps with Next.js 14 and Supabase
+author: jonmeyers_io
+categories:
+  - product
+tags:
+  - nextjs
+  - intgrations
+date: '2023-11-01'
+toc_depth: 3
+image: todo
+thumb: todo
+---
+
+As part of [Next.js Conf 2023](https://nextjs.org/conf), the team at Vercel released [Next.js 14](https://nextjs.org/blog/next-14). The huge headling feature was...
+
+<img
+  className="dark:hidden !m-0"
+  alt="Tweet from Lee Robinson about Next.js 14 containing no new APIs - light mode version"
+  src="/images/blog/2023-11-01-supabase-is-now-compatible-with-nextjs-14/lee-tweet-light.png"
+/>
+
+<img
+  className="hidden dark:block !m-0"
+  alt="Tweet from Lee Robinson about Next.js 14 containing no new APIs - dark mode version"
+  src="/images/blog/2023-11-01-supabase-is-now-compatible-with-nextjs-14/lee-tweet-dark.png"
+/>
+
+That's right, the headline feature is no new features!
+
+This may sound underwhelming at first, but is incredibly good news for the stability of Next.js. This release comes with a huge number of performance and stability improvements - such as [Server Actions](https://nextjs.org/docs/app/api-reference/functions/server-actions) being marked as stable. This means we can finally start promoting this fantastically simple way of authenticating users - entirely server-side!
```

Comment:
```suggestion
This may sound underwhelming at first, but is incredibly good news for the stability of Next.js. This release comes with a huge number of performance and stability improvements - such as [Server Actions](https://nextjs.org/docs/app/api-reference/functions/server-actions) being marked as stable. This means we can finally start promoting this fantastically simple way of authenticating users—entirely server-side!
```